src/constraints.jl

```diff
@@ -129,6 +129,71 @@ function set_dual_start_value(
     return
 end
 
+"""
+    set_start_value(con_ref::ConstraintRef, value)
+
+Set the primal start value (MOI attribute `ConstraintPrimalStart`) of the
+constraint `con_ref` to `value`. To remove a primal start value set it to
+`nothing`.
+
+See also [`start_value`](@ref).
+"""
+function set_start_value(
+    con_ref::ConstraintRef{
+        <:AbstractModel,
+        <:_MOICON{<:MOI.AbstractVectorFunction,<:MOI.AbstractVectorSet},
+    },
+    value,
+)
+    MOI.set(
+        owner_model(con_ref),
+        MOI.ConstraintPrimalStart(),
+        con_ref,
+        vectorize(value, con_ref.shape),
+    )
+    return
+end
+
+function set_start_value(
+    con_ref::ConstraintRef{
+        <:AbstractModel,
+        <:_MOICON{<:MOI.AbstractVectorFunction,<:MOI.AbstractVectorSet},
```

Comment:
Why is this one only for vector constraints ? What happens if you set nothing for scalar constraints ?